aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -292,6 +292,51 @@ module aptos_framework::account {
         account_resource.authentication_key = new_auth_key;
     }
 
+    public entry fun offer_rotation_capability(
+        account: &signer,
+        rotation_capability_sig_bytes: vector<u8>,
+        account_scheme: u8,
+        account_public_key_bytes: vector<u8>,
+        recipient_address: address,
+    ) acquires Account {
+        let addr = signer::address_of(account);
+        assert!(exists_at(addr) && exists_at(recipient_address), error::not_found(EACCOUNT_DOES_NOT_EXIST));
+
+        let account_resource = borrow_global_mut<Account>(addr);
+
+        // proof that this account intends to delegate its rotation capability to another account
+        let proof_challenge = RotationCapabilityOfferProofChallenge {
```

Comment:
Add another field for the address of the source account here.

Reply:
cc @davidiw is this a breaking change?

Reply:
well it isn't being used, so we could try to amend it and see if you can pass the `e2e-move-tests` for upgradeability
it isn't strictly necessary since the public key already ties the proof to the account but it is a good practice

Reply:
just added the address of the source account - do we want to do the same for the `SignerCapabilityOfferProofChallenge` (which would be a breaking change)?